applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -62,7 +63,7 @@ def model_part_reader(self, modelpart, nodeid=0, elemid=0, condid=0):
     def GetMainPath(self):
         return os.getcwd()
 
-    def __init__(self, model, DEM_parameters):
+    def __init__(self, model, DEM_parameters, DEM_material_parameters):
```

Comment:
Now, DEM_material_parameters is not necessary here, am I right?

Reply:
Exactly, now the file name is provided through the parameters. It is much better like this, because changing the number of arguments always gives problems.

Reply:
@maceligueta are you going to change this? I'll wait to add things until so :) This affects me quite heavily hehe

Reply:
It was pushed yesterday.